core/src/test/java/org/apache/accumulo/core/client/ZooKeeperInstanceTest.java

```diff
@@ -33,6 +33,7 @@
 import org.junit.Before;
 import org.junit.Test;
 
+@SuppressWarnings("deprecation")
```

Comment:
Targeted warnings suppressions indicate that a warning has been triaged. Overly broad suppressions, like one on an entire class leave the impression that the individual warnings have not been triaged, and are simply being ignored. Broad suppressions of warnings can mask newly introduced and untriaged warnings. I would prefer we suppress in a more targeted way if we must suppress at all.

Reply:
fixed in d8ccf825b2a377